README.md

```diff
@@ -39,6 +59,22 @@ Run this command in the root of the project directory to install NimblePhxGenTem
 mix do deps.get, deps.compile
 ```
 
+Step 4: Up and running the Phoenix 
+
+Run this command and verify your local website is successfully installed.
+
+```bash
+mix do deps.get, deps.compile
```

Comment:
seem this one is not a command to up and running Phoenix

Also since this template also is including `Mix - Non-Phoenix` project so in that case there is no Phoenix to start, I'm thinking should we removing this step?

Reply:
my typing issue, actually, I meant:

```
mix phx.server
```